marketplace/shopify-sku/src/productResolvers.js

```diff
@@ -29,15 +29,25 @@ export const fetchProductPreviews = async (skus, config) => {
   if (!skus.length) {
     return [];
   }
-  const client = await makeShopifyClient({ parameters: { installation: config } });
-  const query = {
-    first: 250,
-    query: `variants:['sku:${skus.join(' OR ')}']`
-  };
-  const products = await client.product.fetchQuery(query);
-  const variants = productPreviewsToVariantsTransformer(products, skus);
-
-  return variants.map(dataTransformer(config));
+
+  const res = await window.fetch(`https://${config.apiEndpoint}/api/2019-10/graphql`, {
+    method: 'POST',
+    headers: {
+      Accept: 'application/json',
+      'Content-Type': 'application/json',
+      'x-shopify-storefront-access-token': config.storefrontAccessToken
+    },
+    body: JSON.stringify({
+      query: `{nodes (ids: [${skus
+        .map(sku => `"${sku}"`)
```

Comment:
Compute the string w/ IDs outside of this string template

Reply:
Done.